tools/docker/init.sh_centos6

```diff
@@ -17,27 +17,35 @@
 # under the License.
 #
 # update database connection
+# /usr/bin/cloudstack-setup-databases cloud:password@$MYSQL_PORT_3306_TCP_ADDR
 # start cloudstack-management server
-#/usr/bin/cloudstack-setup-databases cloud:password@$MYSQL_PORT_3306_TCP_ADDR
 
 # initial startup of the container to generage ssh_key
 # performed as privileged
 if [ ! -d /var/cloudstack/management/.ssh ]; then
 	mknod /dev/loop6 -m0660 b 7 6
 fi
-sleep 5
 
-mysql -p"$MYSQL_ENV_MYSQL_ROOT_PASSWORD" -h"$MYSQL_PORT_3306_TCP_ADDR" \
+RESTART_REQUIRED=false
+
+until nc -z $MYSQL_PORT_3306_TCP_ADDR 3306; do
```

Comment:
that would be good. giving a hint that the link is probably not there or not set properly....that's why the env does not exist